CHANGELOG.md

```diff
@@ -1,12 +1,14 @@
 # Changelog - Fyne.io fyne-cross
 
-## Unreleased
+## [1.0.0]
 - Add support for "fyne release" #3
 - Add support for creating packaged .tar.gz bundles on freebsd #6
 - Add support for Linux Wayland #10
-- Update fyne cli to v1.4.2-0.20201204171445-8f33697cf611 (fyne-io#1538 fyne-io#1527)
+- Update fyne cli to v1.4.2 (fyne-io#1538 fyne-io#1527)
 - Deprecate `output` flag in favour of `name`
 - Fix env flag validation #14
+- Fix fails to build for linux mobile #19
```

Comment:
Could this perhaps be “Fix build failure for Linux mobile” instead?

Reply:
Thanks for the catch, fixed in b36aee34dc25a7404bef1d0edd6c61efceb257fe